src/Config.hs

```diff
@@ -50,6 +50,7 @@ data ConfigFile = ConfigFile {
 , configFileVersion :: Maybe String
 , configFileSynopsis :: Maybe String
 , configFileDescription :: Maybe String
+, configFileBugReports :: Maybe String
```

Comment:
Could we rename `ConfigFile` in `PackageDescription` and `readConfig` in something appropriate?